### PR TITLE
wire up inspectors

### PR DESCRIPTION
**Pull request by sjmiles**

### CODE REVIEW

File: pkg/AnewLibrary/Design/Services/DesignService.js [hunk 1]
```diff
@@ -198,3 +224,36 @@ const validTarget = (elt, targetId) => {
...
+    [propName]: `${prefixId}$${connection}`
+  };
+  Object.assign(hostConnections, graphQualifiedConnections);
+  designUpdateTarget(controller, host);
+  Project.ProjectService.SaveProject();
+};
+
+const updateProperty = (controller, hostId, propName, value) => {
+  // update controller state and atoms
+  Controller.writeValue(controller, hostId, propName, value);
```
Comment (mariakleiner) [inline pkg/AnewLibrary/Design/Services/DesignService.js:250]: I'll merge and debug.. but just wondering - if the same property has both a value and a connection, what is the expected behaviour?

File: pkg/AnewLibrary/Fields/Atoms/DateField.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  [delim] {
+    padding: 6px;
+  }
+</style>
+
+<div flex bar>
+  <div label>{{label}}</div>
+  <span delim></span>
+  <input flex field value="{{value}}" on-change="onFieldChange" list="options">
+  <datalist id="options" repeat="option_t">{{options}}</datalist>
```
Comment (mariakleiner) [inline pkg/AnewLibrary/Fields/Atoms/DateField.js:64]: I think this file was intended to be called "DataField" not "DateField"?

File: pkg/AnewLibrary/Fields/FieldsNodeTypes.js [hunk 3]
```diff
@@ -0,0 +1,154 @@
+/**
+ * @license
+ * Copyright 2023 Atom54 LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+const category = 'UX';
+
+export const FieldsNodeTypes = {
```
Comment (mariakleiner) [inline pkg/AnewLibrary/Fields/FieldsNodeTypes.js:8]: Is this file needed? I thought this would be replaced by the atom_manifest...

File: pkg/AnewLibrary/Fields/Nodes/BooleanFieldNode.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+/**
+ * @license
+ * Copyright 2023 Atom54 LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+export const BooleanFieldNode = {
```
Comment (mariakleiner) [inline pkg/AnewLibrary/Fields/Nodes/BooleanFieldNode.js:6]: Same question about ...Node.js files - i thought they'd be deprecated?

**Review by mariakleiner — APPROVED** [review]

File: pkg/AnewLibrary/Fields/Atoms/DateField.js (hunk 2 above)
Comment (sjmiles) [reply pkg/AnewLibrary/Fields/Atoms/DateField.js:64]: Is _DateField_ :)

File: pkg/AnewLibrary/Design/Services/DesignService.js (hunk 1 above)
Comment (sjmiles) [reply pkg/AnewLibrary/Design/Services/DesignService.js:250]: It's always been this way, iiuc. Connection wins.

File: pkg/AnewLibrary/Fields/FieldsNodeTypes.js (hunk 3 above)
Comment (sjmiles) [reply pkg/AnewLibrary/Fields/FieldsNodeTypes.js:8]: This is just in progress.

File: pkg/AnewLibrary/Fields/Nodes/BooleanFieldNode.js (hunk 4 above)
Comment (sjmiles) [reply pkg/AnewLibrary/Fields/Nodes/BooleanFieldNode.js:6]: This is just in progress.
